Wikipedia/Code/MWKDataStore.m

```diff
@@ -433,7 +433,7 @@ - (void)performUpdatesFromLibraryVersion:(NSUInteger)currentLibraryVersion inMan
     }
 
     if (currentLibraryVersion < 8) {
-        NSUserDefaults *ud = [NSUserDefaults wmf];
+        NSUserDefaults *ud = [NSUserDefaults standardUserDefaults];
```

Comment:
@joewalsh do you think we should move the library version 10 migration up to before this point?

Reply:
Good catch! I changed this to keep using the wmf defaults since it's before the conversion. I'd be a bit uncomfortable putting the migrations out of order but let me know if there's something I'm missing about that approach

Reply:
@joewalsh cool yeah agreed, feels better to keep the ordering.